docs/application/vstools-mac/overview.md

```diff
@@ -1,150 +1,131 @@
-
-4. Enter the root directory path where you want to install and click **Next**.
-
-   ![Set the installation path](media/howtoinstall-installwizard3.png)
-
-   The Tizen SDK installer is downloaded and the baseline SDK is installed automatically.
-
-   ![Installer download](media/howtoinstall-installwizard4.png)
-
-   ![Baseline SDK installation](media/howtoinstall-installwizard5.png)
-
-   Finally, the Tizen Package Manager installs the Tizen SDK tools.
-
-   ![Tool installation](media/howtoinstall-installwizard6.png)
-
-<a name="set-tizen-baseline-sdk"></a>
-### Set the Tizen Baseline SDK Path 
-
-You can also use the Tizen Package Manager to set up the Tizen baseline SDK path or each tool path directly:
-
-- To set up the Tizen Baseline SDK path:
-  1. In the Visual Studio IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-  2. Select **Use installed Tizen SDK**.
-
-     ![Baseline SDK Install](media/howtoinstall-installwizard7.png)
-
-  3. Enter the root directory of your existing Tizen Studio installation.
-
-     ![Baseline SDK Install](media/howtoinstall-installwizard8.png)
-
-  > **Note**  
-  > If the installer gives a warning about your Tizen Studio version being too low, update Tizen Studio by using the Tizen Package Manager after setting the tool path.
-
-- To set up each tool path directly:
-  1. In the Visual Studio Mac IDE menu, go to **Project &gt; Solution Options &gt; Tizen &gt; Tools**.
-  2. Enter the root directory of your existing Tizen Studio installation in the **Tool Path** field.
-
-     ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
-
-     The other tools paths are automatically set up.
-
-### Install Emulator Images
-
-If you do not have a real device, you can run applications in the Tizen Emulator.
-
-To download emulator images, you can use the Tizen Package Manager or the Tizen Emulator Manager:
-
-- To use the Tizen Package Manager:
-  1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Package Manager**.
-  2. Select the profiles and versions you want to install and click **Install**.
-
-     ![Package Manager](media/howtoinstall-packagemanager.png)
-
-- To use the Tizen Emulator Manager:
-
-  > **Note**
-  >
-  > The Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
-
-  1. In the Visual Studio Mac IDE menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
-  2. Select the profiles and versions you want to install and click **OK**.
-
-     ![Emulator Manager](media/howtoinstall-emulatormanager.png)
-
-
-## Troubleshoot
-
-If you encounter problems with the installation:
-
-- To make sure the baseline Tizen SDK is installed correctly, check the tool path by selecting **Project &gt; Solution Options &gt; Tizen &gt; Tools**.
-
-  ![Check the SDK tool path](media/howtoinstall-checktoolpath.png)
-
-
+# Install Visual Studio for Mac Extension for Tizen.

```

Comment:
@journey2w This page title is in complete sentence form, even with the period mark.
Also, the "Install Visual Studio for Mac Extension for Tizen" is repeated again at line 19.
(The same heading on line 26 is already suggested to be changed.)

I worry in below 2 points : 

- Is it intended to add the period to the title?
- The same sentence is used both to the page title and the section heading on line 19.

For the second issue, how about removing from line 19 to line 24, 
while modifying the line 3 like below?

Visual Studio for Mac Extension for Tizen is an extension, which enables you to develop Tizen .NET applications easily using various Tizen project templates with Visual Studio for Mac. To use the Tizen tools in Visual Studio for Mac, you must install [Visual Studio for Mac Extension for Tizen](#install-the-tizen-extension) and [Tizen Baseline SDK](#install-tizen-baseline-sdk).

Would you please let me know your opinion?

Reply:
@mijin85cho, yes. Removed. Thank you.

Reply:
Please see https://github.com/Samsung/tizen-docs/pull/1017#discussion_r457866161.